### DISCUSSION TITLE
Publish a book in two languages - best approach?

### DISCUSSION THREAD

**Discussion by cylynx11** [quarto-dev/quarto-cli#5735] [Q&A] [answered]
### Description

I will write and release a book (github pages) in two languages (eng, pt_br). Do I need to create two separate repos, one for each language or should I create a single repo and create two main folders within the same project (book), one for each language?

Any help would be greatly appreciated.

**Comment by mcanouil** [comment, marked as answer]
See #275
(Closing the discussion as all discussions/suggestions/_etc._ are on the issue).

---

TL;DR

https://github.com/quarto-dev/quarto-cli/discussions/5262

https://github.com/ropensci-review-tools/babelquarto
